<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Auto Battle Bot - Pkmon</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>911</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_main">
        <attribute name="title">
         <string>Main</string>
        </attribute>
        <widget class="QTableWidget" name="tableWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>401</width>
           <height>421</height>
          </rect>
         </property>
         <column>
          <property name="text">
           <string>id</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Monster</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Row</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Max Round</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Status</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Win</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Loose</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Last Tx</string>
          </property>
         </column>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>420</x>
           <y>470</y>
           <width>61</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Status:</string>
         </property>
        </widget>
        <widget class="QLabel" name="lb_status">
         <property name="geometry">
          <rect>
           <x>480</x>
           <y>480</y>
           <width>111</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Not connected</string>
         </property>
        </widget>
        <widget class="QGroupBox" name="groupBox">
         <property name="geometry">
          <rect>
           <x>420</x>
           <y>0</y>
           <width>471</width>
           <height>80</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>471</width>
           <height>80</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>Control</string>
         </property>
         <widget class="QPushButton" name="btn_connect">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>89</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_start">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>30</y>
            <width>89</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_remove_monster">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>30</y>
            <width>89</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </widget>
        <widget class="Line" name="line">
         <property name="geometry">
          <rect>
           <x>403</x>
           <y>0</y>
           <width>20</width>
           <height>521</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QTextBrowser" name="txt_log">
         <property name="geometry">
          <rect>
           <x>420</x>
           <y>90</y>
           <width>471</width>
           <height>371</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_clear">
         <property name="geometry">
          <rect>
           <x>778</x>
           <y>470</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Clear Log</string>
         </property>
        </widget>
        <widget class="QPlainTextEdit" name="txt_monster_id_update">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>470</y>
           <width>104</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QPlainTextEdit" name="txt_monster_rat_update">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>470</y>
           <width>104</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_update">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>470</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>450</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Monster Id</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>450</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Round</string>
         </property>
        </widget>
        <widget class="Line" name="line_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>430</y>
           <width>411</width>
           <height>16</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_config">
        <attribute name="title">
         <string>Config</string>
        </attribute>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Account</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Private Key</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="txt_acc">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>10</y>
           <width>371</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QTextEdit" name="txt_priv_key">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>50</y>
           <width>371</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="Line" name="line_2">
         <property name="geometry">
          <rect>
           <x>-10</x>
           <y>100</y>
           <width>921</width>
           <height>31</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_save">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>89</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_show">
         <property name="geometry">
          <rect>
           <x>510</x>
           <y>10</y>
           <width>89</width>
           <height>71</height>
          </rect>
         </property>
         <property name="text">
          <string>Show</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Help</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>916</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
